lib/geo_pattern/pattern_store.rb

```diff
@@ -0,0 +1,55 @@
+module GeoPattern
+  class PatternStore
+    private
+
+    attr_reader :store
+
+    public
+
+    def initialize(
+      store = HashStore.new(
```

Comment:
This is kind of a `HashWithIndifferentAccess`. So there's no need to use strings here anymore.